### DISCUSSION TITLE
Is Galaxy Brain counting the current status?

### DISCUSSION THREAD

**Discussion by sleepymalc** [Schweinepriester/github-profile-achievements#135] [Q&A] [answered]
I'm wondering if after getting the achievement, and then we delete all the discussion threads, will the achievement disappear?

**Comment by Schweinepriester** [comment, marked as answer]
Unlikely as the achievements so far seem to be one way (except sponsor), but feel free to test :D

**Reply by sleepymalc** [reply]
Yup, I just tested. 
![image](https://user-images.githubusercontent.com/60427058/179054314-87d06c30-61d2-4cea-a985-c9e7a21f9ec2.png)
The references just become inaccessible, but I still have the achievement.

**Reply by thenithinbalaji** [reply]
oh great

**Reply by sleepymalc** [reply]
So then a theory would be the following: We can mark and unmark one's answer and assign it to another after the initial one gets the achievement.
